lldb/tools/lldb-dap/Handler/ConfigurationDoneRequestHandler.cpp

```diff
@@ -44,6 +44,18 @@ namespace lldb_dap {
 //             just an acknowledgement, so no body field is required."
 //             }]
 // },
+
+llvm::json::Array CacheInitialThreads(DAP &dap) {
```

Comment:
Make this a Utility function in `JSONUtils.h` that takes a `lldb::SBProcess` so we can call it from `ConfigurationDoneRequestHandler::operator()` and from the `threads` handler. So in `JSONUtils.h` make:
```
llvm::json::Array GetThreads(lldb::SBProcess process);
```
And move this function to JSONUtils.cpp